README.md

```diff
@@ -1 +1,22 @@
-# dry-operation
+<!--- this file is synced from dry-rb/template-gem project -->
+[gem]: https://rubygems.org/gems/dry-operation
+[actions]: https://github.com/dry-rb/dry-operation/actions
+
+# dry-operation [![Gem Version](https://badge.fury.io/rb/dry-operation.svg)][gem] [![CI Status](https://github.com/dry-rb/dry-operation/workflows/ci/badge.svg)][actions]
+
+## Links
+
+* [User documentation](https://dry-rb.org/gems/dry-operation)
+* [API documentation](http://rubydoc.info/gems/dry-operation)
+* [Forum](https://discourse.dry-rb.org)
+
+## Supported Ruby versions
+
+This library officially supports the following Ruby versions:
+
+* MRI `>= 3.2.0`
```

Comment:
```suggestion
* MRI `>= 3.0.0`
```

Not sure how this snuck in at 3.2.0 as our minimum; 3.0.0 is where it is currently :)

Reply:
:bow: Amended. Yeah, I was thinking in terms of how I manage all of my gems (with a strong focus on the latest version of Ruby only 😅).